CMakeLists.txt

```diff
@@ -554,15 +560,23 @@ add_definitions(-DWINVER=0x0602)
 add_definitions(-DWIN32_LEAN_AND_MEAN=1)
 add_definitions(-DDEBUGGING_SUPPORTED)
 if(WIN32)
-    # Disable edit and continue on Linux
-    add_definitions(-DEnC_SUPPORTED)
+  if(CLR_CMAKE_PLATFORM_ARCH_AMD64 OR CLR_CMAKE_PLATFORM_ARCH_I386)
+      # Disable edit and continue on Linux, arm & arm64
```

Comment:
I know that this comment was there before, but it sounds weird. Could you please change it to "Enable edit and continue"? We don't need to enumerate the platforms, the surrounding ifs are descriptive enough.